Docs/source/Hardware/Boards.rst

```diff
@@ -20,9 +13,33 @@ the :term:`Shunt Resistor`.
 
     AS8510 Block Diagram 
 
+Fan Board: Manthan Upadhyaya
+=============================
+
+Purpose
+    The purpose of this board is to set the speed of the fans that will be used to cool the battery 
+    pack. It recieves signals from the Master Board and sends them across a :term:`MOSFET <MOSFET>` 
+    that switches the fan on or off.
+
+Blocks
+    Mosfets - Each mosfet is between the power provided by the Master Board and power provided to the
+    fans. When the :term:`PWM <Pulse Width Modulation>` signal is set, it switches the fan on or off 
+    at a certain duty cycle.
+
+Connections
+    6 Connector: 4 of these connections are the PWM signals sent by the Master board. The other 2
+    are power and ground.
+    
+    2 Connector(4): These are the power and grounds each fan is connected to.
+
+Additional Considerations
+    None
+
 Leader board
```

Comment:
The Leader board section seems pretty disorganized to me, but I think fixing it may be beyond the scope of this pull request. What do you think?

Reply:
Yea we can do it in another, or if it is just organizing the information, we can have a newer member do it so they can read a little bit more about the hardware we use. If something is confusing to them, it probably means we would need to add more information.